sharexyz.py

```diff
@@ -594,6 +590,7 @@ def _upload_file(s3_client, file_name, path):
         )
         # mimetypes.add_type('video/mp4', '.mp4')
         file_mime_type, _ = mimetypes.guess_type(file_name)
+        file_mime_type = file_mime_type or "image/webp"
```

Comment:
ig I could've used `mimetypes.add_type('image/webp', '.webp')`?
```suggestion
        mimetypes.add_type('image/webp', '.webp')
        file_mime_type, _ = mimetypes.guess_type(file_name)
```